tests/Behat/Element/Backend/TopBarElement.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat\Element\Backend;
+
+use FriendsOfBehat\PageObjectExtension\Element\Element;
+
+final class TopBarElement extends Element
```

Comment:
I would have to check, but I suppose elements should be without the final keyword and/or should have an interface. 

```suggestion
class TopBarElement extends Element implements TopBarElementInterface
```